setup.py

```diff
@@ -56,7 +56,7 @@ def get_version():
 
         version = scm_get_version()
         # Parse version string (e.g., "1.2.3" or "1.2.3.dev0+g1234567")
-        version_parts = version.split("+")[0].split(".dev")[0].split("rc")[0].split(".")
+        version_parts = version.split("+")[0].split(".dev")[0].split("rc")[0].split("b")[0].split(".")
```

Comment:
What is this about?

Reply:
Dealing with the various pre-release names. eg 1.1.1b0, 1.1.1rc1, etc.